winterfell/test/testMailingService.js

```diff
@@ -56,29 +57,27 @@ describe('Mailing', function () {
     var service;
     var testUserInstance;
     var testDocumentInstance;
-    var testUserAddressInstance;
     var testDestinationAddressInstance;
 
     before(function (done) {
         server = require('../app');
         service = new MailingService(server.app);
         service.Lob = mockLob;
 
-        User.create(testUser, function (userErr, user) {
+        User.remove({}, function() {
```

Comment:
I agree, eventually we should set it up so that then NODE_ENV=test we use a different copy of the database, vs when we run locally